src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
```

Comment:
An element of the parent block MUST NOT be used inside a child block. So add to `li` class `item` and then use `item__link`.
Modifier `is-active` in BEM shoud be used like `item__link--is-active`, but add class `is-active` was in requirements to the task.